apps/server/src/v1/index.ts

```diff
@@ -15,15 +13,16 @@ import { pageSubscribersApi } from "./pageSubscribers";
 import { pagesApi } from "./pages";
 import { statusReportUpdatesApi } from "./statusReportUpdates";
 import { statusReportsApi } from "./statusReports";
+import type { Limits } from "@openstatus/db/src/schema/workspaces/validation";
```

Comment:
Let's consider the `@openstatus/plan` as source of truth for user plan and limits.

```suggestion
import type { Limits } from "@openstatus/plan";
```